classes/class-p4-campaigns.php

```diff
@@ -297,13 +296,57 @@ public function validate_page_types( $selected_page_types ) : bool {
 		 * Load assets.
 		 */
 		public function enqueue_admin_assets() {
-			if ( ! is_admin() || strpos( get_current_screen()->taxonomy, $this->taxonomy ) === false ) {
+			if ( ! is_admin() || strpos( get_current_screen()->post_type, $this->post_type ) === false ) {
```

Comment:
Please revert this. It was probably left from the first commits for the campaigns feature